CMakeLists.txt

```diff
@@ -119,7 +119,8 @@ execute_process(COMMAND git tag --points-at HEAD
         WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
         OUTPUT_VARIABLE GIT_TAG
         ERROR_QUIET)
-if(GIT_TAG)
+string(REGEX REPLACE "latest" "" GIT_TAG "${GIT_TAG}")
+if(GIT_TAG MATCHES "[0-9]+\.[0-9]+\.[0-9]+")
```

Comment:
Did you test this? I could imagine that you could have problems with remaining newlines.

Reply:
It's not an exact match (no ^ $), I tested it with -beta tag. Also we strip newlines a couple lines up if we pull from the git tag.

Reply:
Ah, ok. That's what I was worried about. Not that it wouldn't match, but that the newline might end up in the variable.